Submissions/Gaurav-ECE-SVNIT/Assignment1.py

```diff
@@ -0,0 +1,47 @@
+from hashlib import sha256
+import time
+
+class block():
+	def __init__(self, timestamp, data, previousHash = ''):
+		self.timestamp = timestamp
+		self.data = data
+		self.previousHash = previousHash
+		self.hash = self.calculateHash()
+
+	def calculateHash(self):
+        return sha256((str(self.timestamp)+str(self.data)+str(self.previousHash)).encode()).hexdigest()
```

Comment:
will work..!!

Just to make sure I checked it in Python editor, I found inconsistent use of tabs. please correct it in this line